dev/gpctl/cmd/clusters-register.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/gitpod-io/gitpod/common-go/log"
+	"github.com/gitpod-io/gitpod/ws-manager-bridge/api"
+)
+
+// clustersRegisterCmd represents the clustersListCmd command
+var clustersRegisterCmd = &cobra.Command{
```

Comment:
I'd prefer if the CLI could produce the request, especially because of the cert.
Consider the devops engineer registering the cluster: most likely they'll open a workspace on gitpod-com and run gpctl in there. If they need to duplicate the cert that doesn't sit right. Instead, they could run something like:
```
gpctl clusters register https://example.com/12354325 --name my-new-cluster --cert /workspace/cluster-wsman.crt --hint-cordoned --hint-govern --hint-preferability=prefer
```

You could also (additionally) introduce something like
```
gpctl clusters register --from-file foobar.json --cert /tmp/mycert.crt
```

where the flags override what's in the JSON file.